### PR TITLE
fix: update version

### PR DESCRIPTION
**Pull request by davidbp**

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -5,9 +5,9 @@ description: '`ImageTorchEncoder` encodes data from a `ndarray` into a d-dimensi
 author: Jina AI Dev-Team (dev-team@jina.ai)
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
-version: 0.0.1
```
Comment (nan-wang) [inline manifest.yml:8]: I didn't see the point to have semantic versioning for Jina Hub. Let's use the simplest incremental versioning.
```
version: 2
```

File: tests/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 pytest==6.2.4
 pytest-mock==3.6.1
 pytest-lazy-fixture==0.6.3
-git+https://github.com/jina-ai/jina.git@v2.0.0#egg=jina[hub]
\ No newline at end of file
+git+https://github.com/jina-ai/jina.git@v2.0.0#egg=jina[hub]
```
Comment (nan-wang) [inline tests/requirements.txt]:
```suggestion
git+https://github.com/jina-ai/jina.git@v2.0.3#egg=jina[hub]
```

**Review by nan-wang — CHANGES_REQUESTED** [review]

File: manifest.yml (hunk 1 above)
Comment (davidbp) [reply manifest.yml:8]: ok

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**